layout/layers_ordering_assigner.go

```diff
@@ -58,6 +58,9 @@ func (o WarfieldOrderingOptimizer) Optimize(g Graph, lg LayeredGraph) {
 			copyLayers(bestLayers, layers)
 		}
 		log.Printf("warfield ordering optimizer:\t epoch(%d)\t best(%d)\t current(%d)\n", t, bestN, N)
```

Comment:
better use slog
```suggestion
		slog.Infof("warfield ordering optimizer:\t epoch(%d)\t best(%d)\t current(%d)\n", t, bestN, N)
```better use slogbetter use slog